README.rst

```diff
@@ -40,46 +40,45 @@
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
 .. _NUMFocus: http://www.numfocus.org
 
+.. image:: doc/_static/logo2.png
 
-##########
-Matplotlib
-##########
+Matplotlib is a comprehensive library for creating static, animated, and interactive visualizations in Python. 
 
-Matplotlib is a Python 2D plotting library which produces publication-quality
-figures in a variety of hardcopy formats and interactive environments across
-platforms. Matplotlib can be used in Python scripts, the Python and IPython
-shell (à la MATLAB or Mathematica), web application servers, and various
-graphical user interface toolkits.
+Check out our `home page <http://matplotlib.org/>`_ for more information. 
+
+.. image:: doc/_static/readme_preview.png
 
-NOTE: The current master branch is now Python 3 only.  Python 2 support is
-being dropped.
+Matplotlib produces publication-quality figures in a variety of hardcopy formats
+and interactive environments across platforms. Matplotlib can be used in Python scripts, 
+the Python and IPython shell, web application servers, and various
+graphical user interface toolkits.
 
-`Home page <http://matplotlib.org/>`_
+**Note:** The master branch is Python 3 only.  Python 2 support has been dropped.
```

Comment:
I don‘t think we need this anymore. It‘s 2020 and the default assumption should be Python 3.